data/categories_cuisines.txt

```diff
@@ -2115,6 +2115,12 @@ sk:Vegánske jedlá|Vegetariánske jedlá
 # he:מטבח יפני
 # sk:Japonská kuchyňa
 
+# cuisine-juice
+# en = Juice|Smoothies
```

Comment:
`#` means "commented". @vng do you know why so many cuisines are commented here?

Reply:
A lot of cuisines especially countries-like (French, Italian) interfere with geocoding (https://github.com/organicmaps/organicmaps/commit/cf760b4f24018a649be005b15c3b6d9cc98d5387). I tried to _alive_ this routing [here](https://github.com/organicmaps/organicmaps/pull/3007), but faced with some problems and postponed it.